### ISSUE TITLE
Pixel wise weighting for Cross Entropy Loss

### ISSUE THREAD

**Issue by vgonzalezd** [Project-MONAI/MONAI#6355]
**Is your feature request related to a problem? Please describe.**
Ronneberger et al (Unet)[1], Islam et al[2](Spatially varying label smoothing), and Sugino et al[3](Loss weightings) among others propose to use a **Pixel wise weighting** for the Cross entropy loss. Different from the class imbalance weighting, these "**importance maps**" improve the attention on borders for example.

**Describe the solution you'd like**
We would like: Cross_entropy and Dice_crossEntropy to allow the addition of a pixel-wise map.[https://docs.monai.io/en/stable/_modules/monai/losses/dice.html#DiceCELoss](url)

**Describe alternatives you've considered**
Until now, I and my colleagues modify the ground truth label map before giving it to the network. This implies during the data loader we convert it as one-hot and we preprocess it. 

**Additional context**
We can do the change and submit the merge request if you prefer. We would just like to verify if this is a useful feature. 

Figure 3d[1] and Figure 1b[2]
[1] UNET, Available in: https://arxiv.org/pdf/1505.04597.pdf
[2] Islam, M., & Glocker, B. (2021). Spatially varying label smoothing: Capturing uncertainty from expert annotations. In Information Processing in Medical Imaging: 27th International Conference, IPMI 2021, Virtual Event, June 28–June 30, 2021, Proceedings 27 (pp. 677-688). Springer International Publishing.
Available in: https://arxiv.org/abs/2104.05788
[3]Sugino, T., Kawase, T., Onogi, S., Kin, T., Saito, N., & Nakajima, Y. (2021, July). Loss weightings for improving imbalanced brain structure segmentation using fully convolutional networks. In Healthcare (Vol. 9, No. 8, p. 938). MDPI.
Available in: https://www.researchgate.net/publication/353471270_Loss_Weightings_for_Improving_Imbalanced_Brain_Structure_Segmentation_Using_Fully_Convolutional_Networks

**Comment by wyli** [comment]
please see also the markedloss https://github.com/Project-MONAI/MONAI/blob/dev/monai/losses/spatial_mask.py but it's probably incorrect for many use cases: https://github.com/Project-MONAI/MONAI/issues/4193

**Comment by vgonzalezd** [comment]
This is perfect! It is what I was searching. Thank you so much!
